opal/mca/common/verbs/common_verbs_port.c

```diff
@@ -68,6 +68,14 @@ int opal_common_verbs_port_bw(struct ibv_port_attr *port_attr,
         /* EDR: 25.78125 Gbps * 64/66, in megabits */
         *bandwidth = 25000;
         break;
+    case 64:
+        /* HDR: 50Gbps * 64/66, in megabits */
+        *bandwidth = 50000;
+        break;
+    case 128:
```

Comment:
Hi Jeff,

Thanks for your comments and feedback.
 Regarding the invalid pointer free(), I did some initial debugging and found that openmpi is raising this stack trace whenever it fails to select any of the usable btl for a given process. for example, with my transport if I select btl_openib_flags in such a way where only PUT support is available (flags value 306). I will open a seprate PR for this issue.

Yes, it is true that currently values 64 and 128 are not defined in the kernel/user IB stack. I will push the needed change to the ib-stack very soon.

To the best of my knowledge, those macros are not there in user-space. I can try pushing those to rdma-core once.